functions/tailwind.config.js

```diff
@@ -0,0 +1,877 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Is it necessary to add tailwind config in functions?

Reply:
functions also has a nuxt.config.js file, this file requires tailwind.config as the theme.

Reply:
Firebase will run the following script to copy the file from src
https://github.com/likecoin/liker-land/blob/42aabdb28903175b2700d7c1ef7170a3d263b262/firebase.json#L5